orchestra/interface_api/project_management/views.py

```diff
@@ -29,6 +32,17 @@ def is_project_admin(user):
     return user.groups.filter(name='project_admins').exists()
 
 
+class IsProjectAdmin(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return is_project_admin(request.user)
+
+
+class ProjectList(generics.ListCreateAPIView):
```

Comment:
I don't know how I'd do that on the UX side if they're in a dropdown, but open to suggestions!